src/main/java/vct/col/veymont/WellBehavednessJava.java

```diff
@@ -0,0 +1,85 @@
+package vct.col.veymont;
+
+import java.util.Objects;
+
+public class WellBehavednessJava {
+
+    public static boolean isTau(String label) {
+        return label.equals("true @ Tau");
+    }
+
+    public static boolean checkForwardNonTau(AldebaranTau a) {
+        return a.statesAllMatch(i ->
+
+                // For all (i, label1, j1) and (i, tau^*, j2) ...
+                a.transitionsAllMatch(i, (label1, j1) ->
+                        a.tauClosureAllMatch(i, j2 -> {
+
+                            // There exist (j1, tau^*, k1) and (j2, label2, k2) ...
+                            if (isTau(label1) || a.tauClosureAnyMatch(j1, k1 ->
+                                    a.transitionsAnyMatch(j2, (label2, k2) ->
+
+                                            // Such that:
+                                            Objects.equals(label1, label2) && Objects.equals(k1, k2)))) {
+                                return true;
+                            } else {
+                                System.out.println("(" + i + ",\"" + label1 + "\"," + j1 + ") and (" + i + ",tau^*," + j2 + ")");
+                                System.out.println();
+                                System.out.println(a);
```

Comment:
please use a logging thing, vercors should complain about this.

Reply:
I removed all these print statements